staging/src/k8s.io/client-go/rest/request.go

```diff
@@ -602,7 +602,7 @@ func (r *Request) tryThrottleWithInfo(ctx context.Context, retryInfo string) err
 	if latency > extraLongThrottleLatency {
 		// If the rate limiter latency is very high, the log message should be printed at a higher log level,
 		// but we use a throttled logger to prevent spamming.
-		globalThrottledLogger.Infof(message)
+		globalThrottledLogger.Infof("%s", message)
```

Comment:
isn't there `Info(message)` ?

Reply:
Not at the moment. This particular logger only defines [`Infof`](https://github.com/kubernetes/kubernetes/blob/4925cb66c942b4b4ce3677d343d08f2c5a393fa2/staging/src/k8s.io/client-go/rest/request.go#L666). Since it's not a widely used unexported type, I am hesitant to add more to it, but can easily be done. Let me know if we want to add `Info`.